ember-cli-build.js

```diff
@@ -1,4 +1,4 @@
-/* global require, module */
+/* eslint-env node */
 var EmberAddon = require('ember-cli/lib/broccoli/ember-addon')
```

Comment:
This should be updated to: `const EmberAddon = require('ember-cli/lib/broccoli/ember-addon')`